Error: Could not find or load main class org.apache.maven.surefire.booter.ForkedBooter

Today, I noticed the following test error when packaging OWB from the `master` branch in order to build a Docker image:

```
$ docker image build -t openwayback .

[Output suppressed]

-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Error: Could not find or load main class org.apache.maven.surefire.booter.ForkedBooter

Results :

Tests run: 0, Failures: 0, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for OpenWayback 2.4.0-SNAPSHOT:
[INFO] 
[INFO] OpenWayback ........................................ SUCCESS [  1.611 s]
[INFO] Wayback CDX Server Core Java Classes ............... FAILURE [  3.823 s]
[INFO] Wayback CDX Server Web Application ................. SKIPPED
[INFO] OpenWayback Core Java Classes ...................... SKIPPED
[INFO] OpenWayback Web Application ........................ SKIPPED
[INFO] OpenWayback tar.gz Distribution .................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  5.683 s
[INFO] Finished at: 2018-11-04T18:58:04Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.12.4:test (default-test) on project openwayback-cdx-server: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.12.4:test failed: The forked VM terminated without saying properly goodbye. VM crash or System.exit called ? -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :openwayback-cdx-server
The command '/bin/sh -c mvn package' returned a non-zero code: 1
```

--- a/src/site/markdown/release_notes.md
+++ b/src/site/markdown/release_notes.md
@@ -19,6 +19,7 @@ Full listing of changes and bug fixes are not available prior to release 1.2.0 a
 * Fix toolbar plurality issue when only a single capture. [#372](https://github.com/iipc/openwayback/issues/372)
 * Rewrite `integrity` attribute for resources that implement Subresource Integrity. [#371](https://github.com/iipc/openwayback/issues/371)
 * Upgrade httpclient from 4.3.5. [#380](https://github.com/iipc/openwayback/issues/380)
+* Declare Surefire plugin explicitly due to a change in a default value that breaks builds on newer versions of Maven. [#384](https://github.com/iipc/openwayback/pull/384)
 
 ## OpenWayback 2.3.2 Release
 ### Bug fixes

--- a/wayback-cdx-server-core/pom.xml
+++ b/wayback-cdx-server-core/pom.xml
@@ -11,6 +11,19 @@
     <packaging>jar</packaging>
     <name>Wayback CDX Server Core Java Classes</name>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.22.1</version>
+                <configuration>
+                    <useSystemClassLoader>false</useSystemClassLoader>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
     <dependencies>
         <dependency>
             <groupId>org.netpreserve.commons</groupId>

--- a/wayback-core/pom.xml
+++ b/wayback-core/pom.xml
@@ -26,6 +26,14 @@
 	  </execution>
 	</executions>
       </plugin>
+      <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <version>2.22.1</version>
+          <configuration>
+              <useSystemClassLoader>false</useSystemClassLoader>
+          </configuration>
+      </plugin>
     </plugins>
   </build>
 

